xmlrpc/templates/xmlrpc-info.html

```diff
@@ -4,7 +4,7 @@
 <div class="section">
   <h2>XML-RPC Interface</h2>
   <p>Interface with this page using an XML-RPC client, or use the 
-  <a href="{% url builder %}">interactive builder</a>.
+  <a href="{% url "builder" %}">interactive builder</a>.
```

Comment:
Seems to work. It finds the intended [url](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/urls.py#L29) which leads to the intended [view](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/views.py#L376) which renders the intended [html](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/templates/builder.html).